services/cd-service/pkg/repository/repository.go

```diff
@@ -750,19 +743,31 @@ func (r *repository) ApplyTransformers(ctx context.Context, transformers ...Tran
 	var rev *git.Oid
 	if state.Commit != nil {
 		rev = state.Commit.Id()
+	} else {
+		// first commit in the repo!
```

Comment:
The comment is relevant but I don't like having an empty else statement

Reply:
+1

Reply:
done